packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -5,113 +5,167 @@ import {
     onChoose(choiceId)
   }, [choiceId, onChoose])
+  const handleDelete = useCallback(() => {
+    onDelete(choiceId)
+  }, [choiceId, onDelete])
   const handleKeyDown = useCallback((event) => {
     onKeyDown(choiceId, event)
   }, [choiceId, onKeyDown])
 
   useEffect(() => {
-    if (choiceButton && hasFocus) {
-      choiceButton.current.focus()
+    if (choiceMenuItem && hasFocus) {
+      choiceMenuItem.current.focus()
     }
   })
 
-  let thumbnailHeight = 0
-  if (thumbnailSize === 'small') {
-    thumbnailHeight = 21
-  }
-  if (thumbnailSize === 'medium') {
-    thumbnailHeight = 42
-  }
-  if (thumbnailSize === 'large') {
-    thumbnailHeight = 84
-  }
-
+  const thumbnailHeight = 150
   const thumbnailWidth = Math.round(thumbnailHeight * THUMBNAIL_ASPECT_RATIO)
   const thumbnailSrc = `https://thumbnails.zooniverse.org/${thumbnailWidth}x${thumbnailHeight}/${src.slice(8)}`
   
   return (
-    <Button
-      ref={choiceButton}
-      aria-checked={ariaChecked}
-      disabled={disabled}
+    <StyledBox
+      ref={choiceMenuItem}  
+      // TODO: add the following to translations
+      a11yTitle={`${choiceLabel}` + (selected ? '; identified' : '')}
+      aria-haspopup='true'
+      role='menuitem'
+      align='center'
+      direction='row'
       fill
-      label={
-        <StyledBox
-          align='center'
-          columnsCount={columnsCount}
-          fill
-          forwardedAs='span'
+      onKeyDown={disabled ? DEFAULT_HANDLER : handleKeyDown}
+      pad={{
+        right: '10px',
+        vertical: '5px'
+      }}
+      shadedBackground={shadedBackground}
+      selected={selected}
+      tabIndex={tabIndex}
+    >
+      {selected ? (
+        <DeleteButton
+          choiceLabel={choiceLabel}
+          deleteFn={handleDelete}
+          disabled={disabled}
+          tabIndex={selected && tabIndex === 0 ? 0 : -1}
         >
-          {thumbnailSize !== 'none' && src &&
-            <Image
+          {thumbnailSize === 'none' ? (
+            <Box
+              height='100%'
+              width='40px'
+            />
+          ) : (
+            <StyledImage
               alt=''
-              height='fill'
+              height='50'
               src={thumbnailSrc}
-              width={thumbnailWidth}
-            />}
-          <Text>
-            {choiceLabel}
-          </Text>
-        </StyledBox>
-      }
-      onClick={handleClick}
-      onKeyDown={handleKeyDown}
-      role={role}
-      selected={selected}
-      size='small'
-      tabIndex={tabIndex}
-    />
+              width={thumbnailSize === 'small' ? '50' : '60'}
+            />
+          )}
+        </DeleteButton>
+      ) : null}
+      <StyledButton
+        // TODO: add the following to translations
+        a11yTitle={`Open submenu for ${choiceLabel}`}
+        disabled={disabled}
+        label={
+          <Box
+            align='center'
+            direction='row'
+            forwardedAs='span'
```

Comment:
`forwardedAs` doesn't work here since it's not `styled()`.

```suggestion
            as='span'
```